[23:10:22 [32minfo[39m]: Iniciando y Conectando a la DB
[23:10:22 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:10:22
[23:12:49 [32minfo[39m]: Iniciando y Conectando a la DB
[23:12:49 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:12:49
[23:12:51 [31merror[39m]: uncaughtException: Cannot read properties of undefined (reading 'send')
TypeError: Cannot read properties of undefined (reading 'send')
    at /home/runner/botdiscord/Events/‚ö° Cliente/presencia.js:116:51
[23:12:51 [31merror[39m]: uncaughtException: Cannot read properties of undefined (reading 'send')
TypeError: Cannot read properties of undefined (reading 'send')
    at /home/runner/botdiscord/Events/‚ö° Cliente/presencia.js:116:51
[23:19:09 [32minfo[39m]: Iniciando y Conectando a la DB
[23:19:09 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:19:09
[23:19:11 [31merror[39m]: uncaughtException: Cannot read properties of undefined (reading 'send')
TypeError: Cannot read properties of undefined (reading 'send')
    at /home/runner/botdiscord/Events/‚ö° Cliente/presencia.js:116:51
    at FSReqCallback.oncomplete (node:fs:197:23)
[23:19:11 [31merror[39m]: uncaughtException: Cannot read properties of undefined (reading 'send')
TypeError: Cannot read properties of undefined (reading 'send')
    at /home/runner/botdiscord/Events/‚ö° Cliente/presencia.js:116:51
    at FSReqCallback.oncomplete (node:fs:197:23)
[23:20:02 [32minfo[39m]: Iniciando y Conectando a la DB
[23:20:02 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:20:02
[23:20:04 [31merror[39m]: uncaughtException: client.channels.cache.get(...) is not a function
TypeError: client.channels.cache.get(...) is not a function
    at /home/runner/botdiscord/Events/‚ö° Cliente/presencia.js:116:51
    at FSReqCallback.oncomplete (node:fs:197:23)
[23:20:04 [31merror[39m]: uncaughtException: client.channels.cache.get(...) is not a function
TypeError: client.channels.cache.get(...) is not a function
    at /home/runner/botdiscord/Events/‚ö° Cliente/presencia.js:116:51
    at FSReqCallback.oncomplete (node:fs:197:23)
[23:20:38 [32minfo[39m]: Iniciando y Conectando a la DB
[23:20:38 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:20:38
[23:20:40 [31merror[39m]: uncaughtException: Cannot read properties of undefined (reading 'reply')
TypeError: Cannot read properties of undefined (reading 'reply')
    at /home/runner/botdiscord/Events/‚ö° Cliente/presencia.js:116:51
    at FSReqCallback.oncomplete (node:fs:197:23)
[23:20:40 [31merror[39m]: uncaughtException: Cannot read properties of undefined (reading 'reply')
TypeError: Cannot read properties of undefined (reading 'reply')
    at /home/runner/botdiscord/Events/‚ö° Cliente/presencia.js:116:51
    at FSReqCallback.oncomplete (node:fs:197:23)
[23:22:32 [32minfo[39m]: Iniciando y Conectando a la DB
[23:22:32 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:22:32
[23:26:57 [32minfo[39m]: Iniciando y Conectando a la DB
[23:26:57 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:26:57
[23:29:34 [31merror[39m]: prefix is not defined
[23:37:44 [32minfo[39m]: Iniciando y Conectando a la DB
[23:37:44 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:37:44
[23:37:52 [31merror[39m]: unhandledRejection: The reply to this interaction has already been sent or deferred.
Error [InteractionAlreadyReplied]: The reply to this interaction has already been sent or deferred.
    at ChatInputCommandInteraction.reply (/home/runner/botdiscord/node_modules/discord.js/src/structures/interfaces/InteractionResponses.js:102:46)
    at embedPages (/home/runner/botdiscord/Handlers/Paginas.js:70:42)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.execute (/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:208:9)
[23:37:52 [31merror[39m]: unhandledRejection: The reply to this interaction has already been sent or deferred.
Error [InteractionAlreadyReplied]: The reply to this interaction has already been sent or deferred.
    at ChatInputCommandInteraction.reply (/home/runner/botdiscord/node_modules/discord.js/src/structures/interfaces/InteractionResponses.js:102:46)
    at embedPages (/home/runner/botdiscord/Handlers/Paginas.js:70:42)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.execute (/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:208:9)
[23:43:25 [32minfo[39m]: Iniciando y Conectando a la DB
[23:43:25 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:43:25
[23:43:25 [31merror[39m]: unhandledRejection: Unexpected token '.'
/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:35
                .setDescription(`Hola soy **${client.user.username}** Un bot de Utilidades Multiples.`)
                ^

SyntaxError: Unexpected token '.'
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /home/runner/botdiscord/Handlers/Commands.js:14:25
    at Array.forEach (<anonymous>)
[23:43:25 [31merror[39m]: unhandledRejection: Unexpected token '.'
/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:35
                .setDescription(`Hola soy **${client.user.username}** Un bot de Utilidades Multiples.`)
                ^

SyntaxError: Unexpected token '.'
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /home/runner/botdiscord/Handlers/Commands.js:14:25
    at Array.forEach (<anonymous>)
[23:46:03 [32minfo[39m]: Iniciando y Conectando a la DB
[23:46:03 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:46:03
[23:46:03 [31merror[39m]: unhandledRejection: Unexpected token '.'
/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:35
                .setDescription(`Hola soy **${client.user.username}** Un bot de Utilidades Multiples`)
                ^

SyntaxError: Unexpected token '.'
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /home/runner/botdiscord/Handlers/Commands.js:14:25
    at Array.forEach (<anonymous>)
[23:46:03 [31merror[39m]: unhandledRejection: Unexpected token '.'
/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:35
                .setDescription(`Hola soy **${client.user.username}** Un bot de Utilidades Multiples`)
                ^

SyntaxError: Unexpected token '.'
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /home/runner/botdiscord/Handlers/Commands.js:14:25
    at Array.forEach (<anonymous>)
[23:46:53 [32minfo[39m]: Iniciando y Conectando a la DB
[23:46:53 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:46:53
[23:46:53 [31merror[39m]: unhandledRejection: Unexpected token '.'
/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:35
                .setDescription(`Hola soy **${client.user.username}**`)
                ^

SyntaxError: Unexpected token '.'
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /home/runner/botdiscord/Handlers/Commands.js:14:25
    at Array.forEach (<anonymous>)
[23:46:53 [31merror[39m]: unhandledRejection: Unexpected token '.'
/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:35
                .setDescription(`Hola soy **${client.user.username}**`)
                ^

SyntaxError: Unexpected token '.'
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /home/runner/botdiscord/Handlers/Commands.js:14:25
    at Array.forEach (<anonymous>)
[23:51:58 [32minfo[39m]: Iniciando y Conectando a la DB
[23:51:58 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:51:58
[23:51:58 [31merror[39m]: unhandledRejection: Unexpected token '.'
/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:35
.setDescription(`Hola soy **${client.user.username}** Un bot de Utilidades Multiples.`)
^

SyntaxError: Unexpected token '.'
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /home/runner/botdiscord/Handlers/Commands.js:14:25
    at Array.forEach (<anonymous>)
[23:51:58 [31merror[39m]: unhandledRejection: Unexpected token '.'
/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:35
.setDescription(`Hola soy **${client.user.username}** Un bot de Utilidades Multiples.`)
^

SyntaxError: Unexpected token '.'
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /home/runner/botdiscord/Handlers/Commands.js:14:25
    at Array.forEach (<anonymous>)
[23:52:34 [32minfo[39m]: Iniciando y Conectando a la DB
[23:52:34 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:52:34
[23:52:45 [31merror[39m]: uncaughtException: Received one or more errors
Error: Received one or more errors
    at ArrayValidator.handle (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:473:70)
    at ArrayValidator.parse (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:212:88)
    at EmbedBuilder.addFields (/home/runner/botdiscord/node_modules/@discordjs/builders/dist/index.js:225:31)
    at process.<anonymous> (/home/runner/botdiscord/Plugins/anticrash.js:32:14)
    at process.emit (node:events:525:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
[23:52:45 [31merror[39m]: uncaughtException: Received one or more errors
Error: Received one or more errors
    at ArrayValidator.handle (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:473:70)
    at ArrayValidator.parse (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:212:88)
    at EmbedBuilder.addFields (/home/runner/botdiscord/node_modules/@discordjs/builders/dist/index.js:225:31)
    at process.<anonymous> (/home/runner/botdiscord/Plugins/anticrash.js:32:14)
    at process.emit (node:events:525:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
[23:54:03 [32minfo[39m]: Iniciando y Conectando a la DB
[23:54:03 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:54:03
[23:54:09 [31merror[39m]: uncaughtException: Received one or more errors
Error: Received one or more errors
    at ArrayValidator.handle (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:473:70)
    at ArrayValidator.parse (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:212:88)
    at EmbedBuilder.addFields (/home/runner/botdiscord/node_modules/@discordjs/builders/dist/index.js:225:31)
    at process.<anonymous> (/home/runner/botdiscord/Plugins/anticrash.js:32:14)
    at process.emit (node:events:525:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
[23:54:09 [31merror[39m]: uncaughtException: Received one or more errors
Error: Received one or more errors
    at ArrayValidator.handle (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:473:70)
    at ArrayValidator.parse (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:212:88)
    at EmbedBuilder.addFields (/home/runner/botdiscord/node_modules/@discordjs/builders/dist/index.js:225:31)
    at process.<anonymous> (/home/runner/botdiscord/Plugins/anticrash.js:32:14)
    at process.emit (node:events:525:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
[23:55:23 [32minfo[39m]: Iniciando y Conectando a la DB
[23:55:23 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:55:23
[23:55:51 [31merror[39m]: uncaughtException: Received one or more errors
Error: Received one or more errors
    at ArrayValidator.handle (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:473:70)
    at ArrayValidator.parse (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:212:88)
    at EmbedBuilder.addFields (/home/runner/botdiscord/node_modules/@discordjs/builders/dist/index.js:225:31)
    at process.<anonymous> (/home/runner/botdiscord/Plugins/anticrash.js:32:14)
    at process.emit (node:events:525:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
[23:55:51 [31merror[39m]: uncaughtException: Received one or more errors
Error: Received one or more errors
    at ArrayValidator.handle (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:473:70)
    at ArrayValidator.parse (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:212:88)
    at EmbedBuilder.addFields (/home/runner/botdiscord/node_modules/@discordjs/builders/dist/index.js:225:31)
    at process.<anonymous> (/home/runner/botdiscord/Plugins/anticrash.js:32:14)
    at process.emit (node:events:525:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
[23:57:38 [32minfo[39m]: Iniciando y Conectando a la DB
[23:57:38 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:57:38
[23:57:38 [31merror[39m]: uncaughtException: Received one or more errors
Error: Received one or more errors
    at ArrayValidator.handle (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:473:70)
    at ArrayValidator.parse (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:212:88)
    at EmbedBuilder.addFields (/home/runner/botdiscord/node_modules/@discordjs/builders/dist/index.js:225:31)
    at process.<anonymous> (/home/runner/botdiscord/Plugins/anticrash.js:32:14)
    at process.emit (node:events:525:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
[23:57:38 [31merror[39m]: uncaughtException: Received one or more errors
Error: Received one or more errors
    at ArrayValidator.handle (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:473:70)
    at ArrayValidator.parse (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:212:88)
    at EmbedBuilder.addFields (/home/runner/botdiscord/node_modules/@discordjs/builders/dist/index.js:225:31)
    at process.<anonymous> (/home/runner/botdiscord/Plugins/anticrash.js:32:14)
    at process.emit (node:events:525:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
[23:58:34 [32minfo[39m]: Iniciando y Conectando a la DB
[23:58:34 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:58:34
[23:58:34 [31merror[39m]: unhandledRejection: Unexpected template string
/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:50
`,

SyntaxError: Unexpected template string
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /home/runner/botdiscord/Handlers/Commands.js:14:25
    at Array.forEach (<anonymous>)
[23:58:34 [31merror[39m]: unhandledRejection: Unexpected template string
/home/runner/botdiscord/Commands/üåç Publicos/ayuda.js:50
`,

SyntaxError: Unexpected template string
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /home/runner/botdiscord/Handlers/Commands.js:14:25
    at Array.forEach (<anonymous>)
[23:59:27 [32minfo[39m]: Iniciando y Conectando a la DB
[23:59:27 [32minfo[39m]: Bot Iniciado con Exito Dia de Registro Tue Jan 09 2024 23:59:27
[23:59:37 [31merror[39m]: uncaughtException: Received one or more errors
Error: Received one or more errors
    at ArrayValidator.handle (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:473:70)
    at ArrayValidator.parse (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:212:88)
    at EmbedBuilder.addFields (/home/runner/botdiscord/node_modules/@discordjs/builders/dist/index.js:225:31)
    at process.<anonymous> (/home/runner/botdiscord/Plugins/anticrash.js:32:14)
    at process.emit (node:events:525:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
[23:59:37 [31merror[39m]: uncaughtException: Received one or more errors
Error: Received one or more errors
    at ArrayValidator.handle (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:473:70)
    at ArrayValidator.parse (/home/runner/botdiscord/node_modules/@sapphire/shapeshift/dist/index.js:212:88)
    at EmbedBuilder.addFields (/home/runner/botdiscord/node_modules/@discordjs/builders/dist/index.js:225:31)
    at process.<anonymous> (/home/runner/botdiscord/Plugins/anticrash.js:32:14)
    at process.emit (node:events:525:35)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
